modules/op-release-notes-1-6.adoc

```diff
@@ -0,0 +1,178 @@
+
+[id="new-features-tekton-triggers-0-16-0-release-1-6_{context}"]
+=== Tekton Triggers v0.16.0
+
+* With this update, triggers support the `feature-flags` and `config-defaults` config map to configure feature flags and to set default values respectively. (link:https://github.com/tektoncd/triggers/pull/1182[#1182], link:https://github.com/tektoncd/triggers/pull/1110[#1110])
+
+* This update adds a new metric, `eventlistener_event_count`, that you can use to count events received by the `EventListener` resource. (link:https://github.com/tektoncd/triggers/pull/1160[#1160])
+
+[id="new-features-tekton-triggers-0-15-0-release-1-6_{context}"]
+=== Tekton Triggers v0.15.0
+
+* This update adds `v1beta1` Go API types. With this update, triggers now support the `v1beta1` API version.
++
+With the current release, the `v1alpha1` features are deprecated and are candidates for removal after nine months. Begin using `v1beta1` features instead. (link:https://github.com/tektoncd/triggers/pull/1103[#1103])
+
+[id="new-features-pipelines-operator-1-6_{context}"]
+=== {pipelines-title} Operator
+
+* The current release enables you to configure node selection by specifying the `nodeSelector` and `toleration` values in the `TektonConfig` custom resource (CR). The Operator adds these values to all deployments it creates.
++
+To configure node selection for the Operator's controller and webhook deployment, you edit the `config.nodeSelector` and `config.toleration` fields in the specification for the `Subscription` CR after installing the Operator.
++
+To deploy on an infra node, you update the `TektonConfig` and `Subscription` CRs with the infra label as `nodeSelector` and `toleration` so that these labels are applied to the infra node. (link:https://issues.redhat.com/browse/SRVKP-1544[SRVKP-1544])
+
+[id="deprecated-features-1-6_{context}"]
+== Deprecated features
+
+[id="deprecated-cli-0-21-0-release-1-6_{context}"]
+=== CLI 0.21.0 release
+
+* In CLI 0.21.0, support for all `v1alpha1` resources for `clustertask`, `task`, `taskrun`, `pipeline`, and `pipelinerun` commands are deprecated. These resources will be removed after two releases.
+
+[id="deprecated-tekton-0-16-0-1-6_{context}"]
+=== Tekton Triggers v0.16.0
+
+* In Tekton Triggers v0.16.0, the redundant `status` label is removed from the metrics for the `EventListener` resource. (link:https://github.com/tektoncd/triggers/pull/1166[#1166])
++
+[IMPORTANT]
+====
+Breaking change: The `status` label has been removed from the `eventlistener_http_duration_seconds_*` metric.
+Remove queries that are based on the `status` label.
+====
+
+[id="deprecated-tekton-0-15-0-1-6_{context}"]
+=== Tekton Triggers v0.15.0
+
+* With the current release, the `v1alpha1` features are deprecated and are candidates for removal after nine months. With this update, you can begin using the `v1beta1` Go API types instead. Triggers now supports the `v1beta1` API version. (link:https://github.com/tektoncd/triggers/pull/1103[#1103])
```

Comment:
`candidates for removal after nine months` - the same as above

Reply:
Updated throughout. Thanks.